styles/globals.css

```diff
@@ -25,14 +25,10 @@ html {
   scroll-behavior: smooth;
 }
 
-.homepage-img {
-  background-image: url('/assets/images/main-image.png');
-}
-
-.ch1-background {
-  background-image: url('/assets/images/chapter-2-cover.jpg');
-  background-repeat: no-repeat;
-  background-size: cover;
+.blur {
```

Comment:
I don't see this `blur` class used anywhere in the code (did a search). Is this a leftover or did I miss it?

Reply:
Yes, this is a leftover class. It was used in `layout.tsx` to set the appropriate background properties. But this was replaced with the tailwind theme config in #146.

<img width="1085" alt="image" src="https://user-images.githubusercontent.com/85434418/223082185-4632f028-9657-496e-80a8-7ed005a28fa6.png">

This class can be safely removed.